pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -0,0 +1,144 @@
+package featuremgmt
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"unicode"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFeatureToggleTypeScript(t *testing.T) {
```

Comment:
Nit. A bit weird to do this in a test. Should probably use go generate.

Reply:
my go generate foo is not strong... I see how to get it to create functions from static files, but no idea (and can not find any examples) where it generates typescript and go based on a staticly defined struct.

I added it as a test because we want to fail if things ever get out of sync

Reply:
Okay :smile: We can look into that later if needed. There's probably many examples out on the www. It's just a lite weird that you need to run tests to generate the code and you want to commit the changes. In case you forget that you have to wait for CI to tell you about it